src/libraries/Microsoft.Extensions.Caching.Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.cs

```diff
@@ -126,6 +126,21 @@ public partial class MemoryCacheEntryOptions
         public long? Size { get { throw null; } set { } }
         public System.TimeSpan? SlidingExpiration { get { throw null; } set { } }
     }
+    public partial struct MemoryCacheStatistics
+    {
+// #if NETCOREAPP
+//         public long CurrentEntryCount { get { throw null; } init { throw null; } }
+//         public long? CurrentSize { get { throw null; } init { throw null; } }
+//         public long TotalHits { get { throw null; } init { throw null; } }
+//         public long TotalRequests { get { throw null; } init { throw null; } }
+// #else
+        private int _dummyPrimitive;
+        public long CurrentEntryCount { readonly get { throw null; } set { } }
```

Comment:
>set [](http://example.com/codeflow?start=69&length=3)

I recall in the review decided to not expose the setters. right?

Reply:
If you are going to enable init, please look at the PR [https://github.com/dotnet/runtime/pull/65530/files](https://github.com/dotnet/runtime/pull/65530/files) to see how we handled it and ensure you'll have the typeforward there.

cc @ericstj